pkg/conversion/converter.go

```diff
@@ -495,15 +495,6 @@ func (c *Converter) callCustom(sv, dv, custom reflect.Value, scope *scope) error
 // one is registered.
 func (c *Converter) convert(sv, dv reflect.Value, scope *scope) error {
 	dt, st := dv.Type(), sv.Type()
-	// Apply default values.
-	if fv, ok := c.defaultingFuncs[st]; ok {
```

Comment:
We have unit tests for the default values: https://github.com/kubernetes/kubernetes/blob/025f24aa6d866d5db1a6cfa6079bab5babe6b891/pkg/api/v1/defaults_test.go

All the conversion functions in this file either calls the generated functions or calls the defaulting functions themselves. E.g., https://github.com/kubernetes/kubernetes/blob/025f24aa6d866d5db1a6cfa6079bab5babe6b891/pkg/api/v1/conversion.go#L201